runtime/service.go

```diff
@@ -670,6 +673,20 @@ func (s *service) startVM(ctx context.Context,
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to build VM options")
 	}
+
+	// Stub out drives that will be eventually replaced with actual drives.
+	tempDir := os.TempDir()
+	if tempDir, err = ioutil.TempDir(tempDir, "stub-drives"); err != nil {
```

Comment:
nit: no need to call `os.TempDir` explicitly: https://github.com/golang/go/blob/131eb8fbf80fd8b51ae8b5c5220d566582a41e71/src/io/ioutil/tempfile.go#L88

Reply:
I don't like to use magic values as it makes the code harder to follow. In this case the reader needs to know that `""` maps to `os.TempDir`.